packages/react-components/react-combobox/src/components/Dropdown/renderDropdown.tsx

```diff
@@ -26,7 +26,7 @@ export const renderDropdown_unstable = (state: DropdownState, contextValues: Dro
           (state.inlinePopup ? (
             <slots.listbox {...slotProps.listbox} />
           ) : (
-            <Portal>
+            <Portal {...slotProps.portal}>
```

Comment:
I wanted to use slots.portal here, but Portal is a pure function and not a ForwardRef component, so I get this error:

JSX element type 'slots.portal' does not have any construct or call signatures.ts(2604)